test/test_um2netcdf.py

```diff
@@ -148,7 +156,7 @@ def test_process_no_heaviside_drop_cubes(air_temp_cube, precipitation_flux_cube,
 
         # TODO: lat/long & level coord fixes require more internal data attrs
         #       skip temporarily to manage test complexity
-        mock.patch("umpost.um2netcdf.fix_latlon_coord"),
+        mock.patch("umpost.um2netcdf.fix_latlon_coords"),
```

Comment:
There's an opportunity to remove these mocks when the new & improved `fix_latlon_coords()` is merged. This could require modifying the test data, but the real function is called.

_Democking_ this case is probably better in another smaller PR.